scripts/assets.js

```diff
@@ -49,6 +55,13 @@ export async function searchAsset(programName, campaignName, imageWidth = 80) {
 
   const indexName =  await getSearchIndex();
 
+
+  if (programName==null && campaignName==null)
```

Comment:
This is effectively a falsy check but looks like it is checking for null. I'd prefer that this is changed to `if (!progamName && !campaginName)`.

Alternatively, this could be a `===` check, but then the if statements below should also be changed to check `!== null`. Either change works for me.

Reply:
cc: @TyroneAEM